assets/less/epub.less

```diff
@@ -17,6 +17,10 @@ nav > ol {
   }
 }
 
+img[src*="#gh-light-mode-only"] {
```

Comment:
Shouldn't we keep the light one for EPUB and hide the dark one instead?

Reply:
🤦🏻 ...